xdsl/irdl/declarative_assembly_format.py

```diff
@@ -271,6 +271,8 @@ def is_present(self, op: IRDLOperation) -> bool:
     def is_anchorable(self) -> bool:
         """
         Can appear as an anchor in an optional group.
+        Anchorable directives that are not optional-like can't appear as the first
```

Comment:
I feel this has nothing to do with being anchorable?

Reply:
You're right I moved the comment.